### DISCUSSION TITLE
Upgrade to v 0.108.0 [Jul 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6051] [Discord Threads]
> 🙋‍♂️ **_browse**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Yes. I would recommend bug fix release instead though   
#### https://github.com/nocodb/nocodb/releases/tag/0.108.1  
#  
> 🙋‍♂️ **_browse**  
#### Thanks for your reply.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1115175067811323964)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
